### PR TITLE
Clean up dao and add tests

### PR DESCRIPTION
**Pull request by Seeker1911**

### CODE REVIEW

**Comment by Seeker1911** [comment]
after our discussion over resending orders this ended up being a minor cleanup and adding a couple more tests, one that mocks out the `ServiceUnavailable` from Biobank.

File: rest-api/test/unit_test/dao_test/dv_order_dao_test.py [hunk 1]
```diff
@@ -55,3 +59,31 @@ def test_insert_biobank_order(self):
...
+
+  def test_enumerate_tracking_status(self):
+    fhir_resource = SimpleFhirR4Reader(self.post_delivery)
+    status = self.dao._enumerate_order_tracking_status(fhir_resource.extension.get(url=VIBRENT_FHIR_URL + 'tracking-status').valueString)
+    self.assertEquals(status, OrderShipmentTrackingStatus.ENROUTE)
+
+  @mock.patch('dao.dv_order_dao.MayoLinkApi')
+  def test_service_unavailable(self, mocked_api):
+    def raises(*args):
+      print args
```
Comment (rennat) [inline rest-api/test/unit_test/dao_test/dv_order_dao_test.py]: we should remove the extra `print`

Comment (Seeker1911) [reply rest-api/test/unit_test/dao_test/dv_order_dao_test.py]: I agree, but I could not get #pylint disable=unused-argument to work. I put it on every line ! Hence, the "still trying" commit comments above. Doing this was the only thing I could find to shut it up 🤷🏻‍♂️

Comment (rennat) [reply rest-api/test/unit_test/dao_test/dv_order_dao_test.py]: Try one with a `:` after `#pylint`, if it still complains I'll approve as-is

```suggestion
      #pylint: disable=unused-argument
```

Comment (Seeker1911) [reply rest-api/test/unit_test/dao_test/dv_order_dao_test.py]: 🙄done, thanks.

**Pull request merged**